bittensor/metagraph.py

```diff
@@ -18,12 +18,49 @@
 
 from bittensor.exceptions.handlers import rollbar
 
+MAX_INT_WEIGHT = 4294967295 # Max weight value on chain.
+
 def int_to_ip(int_val):
     return str(netaddr.IPAddress(int_val))
  
 def ip_to_int(str_val):
     return int(netaddr.IPAddress(str_val))
 
+class SubscribeSuccess (Exception):
+    """ Raised when try_async_subscribe successfully subscribes to the chain. """
+    pass
+
+class SubscribeUnknownError(Exception):
+    """ Raised when try_async_subscribe throws an unknown error. """
+    pass
+
+class SubscribeTimeout (Exception):
+    """ Raised when try_async_subscribe times out trying to subscribe to chain. """
+    pass
+
+class EmitSuccess (Exception):
+    """ Raised when try_async_emit emits successfully with known result. """
+    pass
+
+class EmitNoOp(Exception):
+    """ Raised when try_async_emit has no weights to change on chain. """
+    pass
+
+class EmitUnknownError (Exception):
+    """ UnknownError error thrown during try_async_emit. """
+    pass
+
+class EmitValueError (Exception):
+    """ Raised during try_async_emit when passed weights are not properly set."""
+    pass
+
+class EmitTimeoutError (Exception):
+    """ Raised during try_async_emit after a wait_for_inclusion timeout. """
+    pass
+
+class EmitResultUnknown(Exception):
+    """ Raised when try_async_emit returns without known result, notably when the call has wait_for_inclusion = False """
+    pass
```

Comment:
Are these exceptions? Some of them appear to be non-erroneous, like `SubscribeSuccess`..

Should move this to the `Exceptions.py` class we have.

Reply:
Keeping it local for now while in active dev. They are acting more like Dispatch Results than actual errors.